docs/Release Notes/release-notes-2023.md

```diff
@@ -3,6 +3,50 @@ title: 2023
 description: "Paralus release history with release notes for 2023"
 slug: /release-notes/2023
 ---
+---
+## v0.2.6
+
+**September 25, 2023**
+
+What changed as part of this release ?
+
+We have made significant improvements to auditlogs, with v0.2.6 ability to view audit logs has been extended to non organization admin users thus improving productivity and giving more insights to users with inbuilt project, cluster admins and read only roles, even custom roles with relavent access permissions.
+
+**Breaking changes**
+
+Prior to paralus v0.2.5, users will not have org, partner metadata information in kratos identities which will impact audit logs screens, apply below migrations if you are upgrading paralus
+
+`update identities set metadata_public = jsonb_set(metadata_public, '{organization}', '"replace-with-your-organization-id"', true);`
```

Comment:
Let's add a new line after this, currently this and next line are rendered as paragraph which is hard to read.

Reply:
thanks for catching this.